tests/README.md

```diff
@@ -205,3 +205,17 @@ You can do that by putting following lines in your `$HOME/.ssh/config` file:
 StrictHostKeyChecking no
 UserKnownHostsFile /dev/null
 ```
+
+## Analysis of results
+
+The rule tests results are saved as `results.json` into the corresponding log directory.
+You can then analyze those results by running e.g.
+
+```
+python analyze_results.py $(find . -name results.json)
```

Comment:
`python3`?

Reply:
Both python 2 and 3 are supported, so I would leave it like this.